packages/swapper/src/swappers/JupiterSwapper/JupiterSwapper.ts

```diff
@@ -3,10 +3,34 @@ import type { Asset } from '@shapeshiftoss/types'
 
 import { executeSolanaTransaction } from '../..'
 import type { BuyAssetBySellIdInput, Swapper } from '../../types'
+import { JUPITER_ERRORS, SolanaLogsError } from './errorPatterns'
 import { jupiterSupportedChainIds } from './utils/constants'
 
 export const jupiterSwapper: Swapper = {
-  executeSolanaTransaction,
+  executeSolanaTransaction: async (...args) => {
+    try {
+      const txid = await executeSolanaTransaction(...args)
+      return txid
+    } catch (e) {
+      if (e instanceof Error) {
```

Comment:
remove me, all errors are `instanceof Error`

Reply:
But it's needed to make TS happy, in order to consume e.message

Reply:
As it's been stamped, closing this to avoid getting this PR stale, feel free to slap me if there is a better way